Server/routing/index.js

```diff
@@ -27,39 +27,45 @@ router.get('/', async (ctx) => {
 
 router.get('/t/:miniURL', async (ctx) => {
     // find url in DB or error- does not exist
+    let id = parseInt(ctx.params.miniURL, 36);
+    console.log(id);
+    try {
+        var query = await database.query(
+            'SELECT * FROM public.urls WHERE id = ' + id
+        )
+
+        
+        if (query.rows[0] != null) {
+            await database.query(
+                'UPDATE public.urls SET counter = counter + 1 WHERE id = ' + id
```

Comment:
See my other comment about using parameterized queries, but if somewhere else you _did_ need to interpolate values into a string, it's easier in modern JS to do (note the use of backticks instead of normal single- or double-quotes):
```
`UPDATE public.urls SET counter = counter + 1 WHERE id = ${id}`
```